src/OXRS_Rack32.cpp

```diff
@@ -588,7 +594,7 @@ void OXRS_Rack32::_updateTempSensor(void)
     if (temperature != NAN)
```

Comment:
Does this need to change to `isnan(temperature)`?

Reply:
good catch. Sure it has to. I overlooked that.
I'll add this tothe PR

Reply:
Should I now also re-revert this one to get temperature display early at startup?
```
  
  // Set the temp sensor resolution (higher res takes longer for reading)
  _tempSensor.setResolution(MCP9808_MODE);
  _lastTempUpdate = -_tempUpdateMs;
}

```
set ` _lasTempUpdate `

Then this PR would solve  #41 

Or di you have a different plan?